Add identifier to allow the modification of the nodes (#4562)
rotki/rotki: docs/api.rst

      Content-Type: application/json

      {
        "identifier": 8,
        "name": "my_node",
        "endpoint": "http://localhost:8386",
        "owned": true,
        "weight": 80,
        "active": false
      }

>    :resjson int identifier: Id of the node that wants to be edited.

Reviewer: ```suggestion
   :resjson int identifier: Id of the node that will be edited.
```

      Content-Type: application/json

      {
        "identifier": 8
      }

>    :resjson int identifier: Id of the node that wants to be deleted.

Reviewer: ```suggestion
   :resjson int identifier: Id of the node that will be deleted.
```
